db/post_migrate/20210628035905_drop_duration_column_from_topic_timers.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class DropDurationColumnFromTopicTimers < ActiveRecord::Migration[6.1]
+  DROPPED_COLUMNS ||= {
+    topic_timers: %i{duration}
```

Comment:
I am not sure whether I remove this from the model in this PR or the next one? I am guessing I need another PR, because otherwise it will un-ignore the column before it is dropped?

https://github.com/discourse/discourse/blob/825b247639cbe34d0d4b32dc9e220e12ebaab33a/app/models/topic_timer.rb#L6-L8

Reply:
Tricky... if it has already been ignored, we can technically drop this in a normal migration without any downsides but I don't think our migration system allows that.

Reply:
@tgxworld yeah I already tried dropping in a normal migration but it stopped me :( Guess I will have to open another PR after this